bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/contentfragment/ContentFragmentImpl.java

```diff
@@ -146,31 +137,31 @@ public String getGridResourceType() {
     @NotNull
     @Override
     public String getEditorJSON() {
-        return getDAMContentFragment().getEditorJSON();
+        return damContentFragment.getEditorJSON();
     }
 
     @NotNull
     @Override
     public Map<String, DAMContentElement> getExportedElements() {
-        return getDAMContentFragment().getExportedElements();
+        return damContentFragment.getExportedElements();
     }
 
     @NotNull
     @Override
     public String[] getExportedElementsOrder() {
-        return getDAMContentFragment().getExportedElementsOrder();
+        return damContentFragment.getExportedElementsOrder();
     }
 
     @Nullable
     @Override
     public List<DAMContentElement> getElements() {
-        return getDAMContentFragment().getElements();
+        return damContentFragment.getElements();
     }
 
     @Nullable
     @Override
     public List<Resource> getAssociatedContent() {
-        return getDAMContentFragment().getAssociatedContent();
+        return damContentFragment.getAssociatedContent();
```

Comment:
I'd also replace `getDAMContentFragment()` with `damContentFragment` in the `getParagraphs()` method.